gems/quilt_rails/lib/quilt_rails/performance/reportable.rb

```diff
@@ -4,8 +4,12 @@ module Quilt
   module Performance
     module Reportable
       def process_report(&block)
-        params.transform_keys! { |key| key.underscore.to_sym }
```

Comment:
We seem to already do this in [`Report.from_params`](https://github.com/Shopify/quilt/blob/e00ab07c32658211eac3c27b31cba87f24d5cfc7/gems/quilt_rails/lib/quilt_rails/performance/report.rb#L11)

Reply:
That makes sense.